packages/orchestrator-ui-components/src/components/WfoTable/WfoTable/WfoGroupedTable/utils.ts

```diff
@@ -22,14 +24,17 @@ export const groupData = <T>(
     );
 
     if (groupByFunctions.length <= 1) {
-        return groupedData;
+        return sortObjectByProperties(groupedData);
     }
 
-    const entries = Object.entries(groupedData).map(([key, value]) => [
-        key,
-        groupData(value, groupByFunctions.slice(1)),
-    ]);
-    return Object.fromEntries(entries);
+    const entries = Object.entries(groupedData).map(([key, value]) => {
+        return [key, groupData(value, groupByFunctions.slice(1))];
+    });
+    return sortObjectByProperties(Object.fromEntries(entries));
+};
+
+export const sortObjectByProperties = (object: object) => {
```

Comment:
Since this function returns a new object, consider renaming to `toObjectWithSortedPropperties`

Reply:
done